### DISCUSSION TITLE
Why?

### DISCUSSION THREAD

**Discussion by ghost** [pi0/config-dir#10] [General]
This proposal is silly, more standards, a new one and an old one to make configuration even more confusing is completely superfluous. `.config` is the way it is because it allows users and developers more freedom when developing software.

**Comment by pi0** [comment]
Hi. I respect your feedback. But maybe you shouldn't describe a proposal _as silly_ because there is something already there or when you simply disagree with it ;) 

The description of the proposal is clear, there are **no standards** for config placement in projects and that's a fact. And yes, the proposal recommends `.config` dir as **opt-in** to give exactly that _freedom_.

**Reply by ghost** [reply]
Fair enough but wouldn't you say there already is a "standard", in that, most applications on, for example, \*Nix systems, put their config files inside `.config` anyway? Usually as `.config/app_name/app_name.{toml,conf.rc}` etc.

**Reply by ghost** [reply]
And forgive me for calling it silly, but I used that word because most systems have enough bloat these days, I think we should strive to KISS it.

**Reply by ghost** [reply]
(Keep-It-Simple-And-Stupid)

**Reply by pi0** [reply]
Yes and exactly config-dir is inspired from XDG/Nix `.config` dir that was made for OS to be consistent.

This proposal aims for different areas BTW. It is not for system-wide configurations. It is for configurations that will be "version-controlled" in code repos.

Using `.config/[name]` is also a good pattern and actually, we mentioned this too in the "config nesting" section. Only the problem is that the ecosystem is likely to reject if we require nested dir. As most of the version-controlled files are one file today and they mostly need to be "organized" on one place. Which as you again also suggested simplest method which is top of `<project>/.config/`)
